Assets/Scripts/Enemy/WanderState.cs

```diff
@@ -0,0 +1,57 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.AI;
+
+public class WanderState : State
+{
+    public GameObject[] waypoints;
+
+    public GameObject Lantern;
```

Comment:
```suggestion
    public GameObject lantern;
```
Conventie voor variabelen is camelCasing